frontend/src/styles/organisations.css

```
@@ -0,0 +1,102 @@
+        border-radius: 5px;
+        font-size: 16px;
+      }
+      
+      .sort-dropdown {
+        padding: 8px;
+        font-size: 16px;
+        border-radius: 5px;
+        border: 2px solid #ccc;
+        cursor: pointer;
+      }
+      t-family: Arial, sans-serif;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)